content-src/components/DiscoveryStreamComponents/DSMessage/DSMessage.jsx

```diff
@@ -17,11 +18,13 @@ export class DSMessage extends React.PureComponent {
             />
           )}
           {this.props.title && (
-            <span className="title-text">{this.props.title}</span>
+            <span className="title-text">
+              <FluentOrText message={this.props.title} />
+            </span>
           )}
           {this.props.link_text && this.props.link_url && (
             <SafeAnchor className="link" url={this.props.link_url}>
-              {this.props.link_text}
+              <FluentOrText message={this.props.link_text} />
```

Comment:
These accept a text value, and it'll then just display the text value, or a Fluent object which then does the proper locale value lookup.